test/jdk/java/lang/StackWalker/StackStreamTest.java

```diff
@@ -214,8 +214,8 @@ static void firstFrame() {
     }
 
     private static <T> void equalsOrThrow(String label, List<T> list, List<T> expected) {
-        System.out.println("List:    " + list);
-        System.out.println("Expectd: " + list);
+        System.out.println("List:     " + list);
+        System.out.println("Expected: " + list);
```

Comment:
I think there is a per-existing bug here. Shouldn't the second println print `expected` instead of `list`?